composer.lock

```diff
@@ -2614,16 +2620,16 @@
         },
         {
             "name": "symfony/yaml",
-            "version": "v4.4.7",
+            "version": "v4.4.8",
```

Comment:
I wonder what dependency still does not support v5.0 ... hm

Reply:
Run a `composer why symfony/yaml` to find out...

Reply:
```mnapoli/front-yaml      1.6.0       requires  symfony/yaml (~2.1|^3.0|^4.0)  ```

Last release `29 Oct 2017`, last update also 3 years ago.

Would be nice if we can get rid of it...

<img width="524" alt="image" src="https://user-images.githubusercontent.com/7423207/80706242-e1de3e00-8adf-11ea-9f90-ce93f0e314a3.png">

Reply:
Maybe: https://github.com/spatie/yaml-front-matter

Reply:
yeah, I was just looking on it, and it's small and simple library, just doing what we need :)